pytensor/graph/basic.py

```diff
@@ -970,6 +970,136 @@ def applys_between(
     )
 
 
+def truncated_graph_inputs(
+    outputs: Sequence[Variable], conditions: Optional[Collection[Variable]] = None
```

Comment:
```suggestion
    outputs: Sequence[Variable], ancestors_to_include: Optional[Collection[Variable]] = None
```

Reply:
Also a lot of "condition" being mentioned in the docstrings / comments that may be better updated.

Reply:
oh, I like the naming idea